index.html

```diff
@@ -644,9 +645,8 @@ <h1>How Can I Participate in FOSSASIA projects?</h1>
 </html>
 
 <style>
-#results_projects {
```

Comment:
I don't think this has anything related to issue

Reply:
yes it is not related to the issue. i guess have re written the 647th line thats why it is in red.i m just removing those extra spaces.

Reply:
yup, this line was already present.this arrised because i left an empty line after 647. i have removed it in latest commit.